utilities/pytest_utils.py

```
@@ -133,12 +133,21 @@ def config_default_storage_class(session):
             for sc in py_config["storage_class_matrix"]
             for sc_name, sc_dict in sc.items()
             if sc_name == updated_default_sc
-        ][0]
+        ]
+
+        if not matching_configurations:
+            available_sc_names = [sc_name for sc in py_config.get("system_storage_class_matrix", []) for sc_name in sc]
+            raise ValueError(
+                f"Storage class '{updated_default_sc}' not found in storage_class_matrix. "
+                f"Available storage classes: {available_sc_names}"
+            )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)